### PR TITLE
Fix quaternion logging

### PR DESCRIPTION
**Pull request by harshil21**

### CODE REVIEW

File: airbrakes/imu/imu.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ class IMU:
 
     def __init__(self, port: str, frequency: int, upside_down: bool):
         # Shared Queue which contains the latest data from the IMU
-        self._data_queue: multiprocessing.Queue[IMUDataPacket] = multiprocessing.Queue()
+        self._data_queue: multiprocessing.Queue[IMUDataPacket] = multiprocessing.Queue(maxsize=100000)
```
Comment (JacksonElia) [inline airbrakes/imu/imu.py:39]: make this a constant

File: airbrakes/imu/imu.py [hunk 2]
```diff
@@ -101,19 +101,18 @@ def _fetch_data_loop(self, port: str, frequency: int, _: bool):
...
                         # https://github.com/LORD-MicroStrain/MSCL/blob/master/MSCL/source/mscl/MicroStrain/MIP/MipTypes.cpp
                         # Check if the imu_data_packet has an attribute with the name of the channel
-                        if hasattr(imu_data_packet, channel):
+                        if hasattr(imu_data_packet, channel) or "Quaternion" in channel:
                             # First checks if the data point needs special handling, if not, just set the attribute
                             match channel:
                                 # These specific data points are matrix's rather than doubles
                                 case "estAttitudeUncertQuaternion" | "estOrientQuaternion":
                                     matrix = data_point.as_Matrix()
                                     # Converts the [4x1] matrix to the X, Y, Z, and W of the quaternion
```
Comment (JacksonElia) [inline airbrakes/imu/imu.py:110]: delete comment

**Review by JacksonElia — APPROVED** [review]
A few small changes

File: airbrakes/imu/imu.py (hunk 2 above)
Comment (harshil21) [reply airbrakes/imu/imu.py:110]: you mean delete this "Check if the imu_data_packet has an attribute with the name of the channel"?

Comment (JacksonElia) [reply airbrakes/imu/imu.py:111]: nah, I made the change I was talking about

**Review by JacksonElia — APPROVED** [review]

**Pull request merged**